src/core_plugins/kibana/public/visualize/listing/visualize_listing.html

```diff
@@ -39,7 +39,7 @@
           ng-click="listingController.deleteSelectedItems()"
           tooltip="Delete selected visualizations"
         >
-          <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-trash"></span>
+          <trash-icon-component></trash-icon-component>
```

Comment:
I like that idea, but keep in mind not all react components will be coming from the ui framework.  I think that's a great idea for ui framework components, but what about other components?  E.g. `<visualize-landing-table></visualize-landing-table>`